recipes/wildmagic/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+cd WildMagic5
+
+export CFG="ReleaseDynamic"
+make -f makefile.wm5
+
+mkdir -p $PREFIX/include/libwildmagic/
+cp -r SDK/Include/* $PREFIX/include/libwildmagic
+cp -r SDK/Library/$CFC/* $PREFIX/lib
+
+mkdir -p $PREFIX/share/libwildmagic
+cp -r Data $PREFIX/share/libwildmagic
```

Comment:
The data dir contains a couple of images and (I think) meshes, that can be used by applications build on wildmagic.